maximum recursion depth exceeded while getting the str of an object

# ComfyUI Error Report
## Error Details
- **Node ID:** 25
- **Node Type:** LoadInvSRModels
- **Exception Type:** RecursionError
- **Exception Message:** maximum recursion depth exceeded while getting the str of an object
## Stack Trace
```
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 349, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 224, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 196, in _map_node_over_list
    process_inputs(input_dict, i)

  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 185, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR\node.py", line 95, in loadmodel
    base_sampler = BaseSampler(configs)
                   ^^^^^^^^^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR\comfyui_invsr_trimmed\sampler_invsr.py", line 42, in __init__
    self.build_model()

  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR\comfyui_invsr_trimmed\sampler_invsr.py", line 90, in build_model
    sd_pipe.vae._set_gradient_checkpointing(sd_pipe.vae.encoder, True)

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\diffusers\models\modeling_utils.py", line 1718, in _set_gradient_checkpointing
    logger.debug(f"Setting `gradient_checkpointing={enable}` for '{name}'")
                                                   ^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^

  [Previous line repeated 740 more times]

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\container.py", line 369, in __repr__
    list_of_reprs = [repr(item) for item in self]
                     ^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\container.py", line 369, in __repr__
    list_of_reprs = [repr(item) for item in self]
                     ^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2934, in __repr__
    extra_repr = self.extra_repr()
                 ^^^^^^^^^^^^^^^^^

  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\normalization.py", line 316, in extra_repr
    return "{num_groups}, {num_channels}, eps={eps}, " "affine={affine}".format(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```
## System Information
- **ComfyUI Version:** 0.3.34
- **Arguments:** C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\main.py --user-directory C:\Users\Administrator\Documents\ComfyUI\user --input-directory C:\Users\Administrator\Documents\ComfyUI\input --output-directory C:\Users\Administrator\Documents\ComfyUI\output --front-end-root C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\web_custom_versions\desktop_app --base-directory C:\Users\Administrator\Documents\ComfyUI --extra-model-paths-config C:\Users\Administrator\AppData\Roaming\ComfyUI\extra_models_config.yaml --log-stdout --listen 127.0.0.1 --port 8002
- **OS:** nt
- **Python Version:** 3.12.9 (main, Feb 12 2025, 14:52:31) [MSC v.1942 64 bit (AMD64)]
- **Embedded Python:** false
- **PyTorch Version:** 2.6.0+cu126
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
  - **Type:** cuda
  - **VRAM Total:** 25769279488
  - **VRAM Free:** 19535588692
  - **Torch VRAM Total:** 5167382528
  - **Torch VRAM Free:** 277441876

## Logs
```
2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - 2025-05-22 19:29:46.6512025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** Platform:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - Windows2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** Python version:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - 3.12.9 (main, Feb 12 2025, 14:52:31) [MSC v.1942 64 bit (AMD64)]2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** Python executable:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - C:\Users\Administrator\Documents\ComfyUI\.venv\Scripts\python.exe2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** ComfyUI Path:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** ComfyUI Base Folder Path:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** User directory:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - C:\Users\Administrator\Documents\ComfyUI\user2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** ComfyUI-Manager config path:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - C:\Users\Administrator\Documents\ComfyUI\user\default\ComfyUI-Manager\config.ini2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:46.651137 - ** Log path:2025-05-22T19:29:46.651137 -  2025-05-22T19:29:46.651137 - C:\Users\Administrator\Documents\ComfyUI\user\comfyui.log2025-05-22T19:29:46.651137 - 
2025-05-22T19:29:48.020113 - [ComfyUI-Manager] Failed to restore comfyui-frontend-package
2025-05-22T19:29:48.020113 - expected str, bytes or os.PathLike object, not NoneType
2025-05-22T19:29:48.021110 - 
Prestartup times for custom nodes:
2025-05-22T19:29:48.021110 -    3.0 seconds: C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\custom_nodes\ComfyUI-Manager
2025-05-22T19:29:48.021110 - 
2025-05-22T19:29:49.309753 - Checkpoint files will always be loaded safely.
2025-05-22T19:29:49.406062 - Total VRAM 24576 MB, total RAM 130953 MB
2025-05-22T19:29:49.406062 - pytorch version: 2.6.0+cu126
2025-05-22T19:29:49.406062 - Set vram state to: NORMAL_VRAM
2025-05-22T19:29:49.406062 - Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
2025-05-22T19:29:50.400152 - Using pytorch attention
2025-05-22T19:29:51.435502 - Python version: 3.12.9 (main, Feb 12 2025, 14:52:31) [MSC v.1942 64 bit (AMD64)]
2025-05-22T19:29:51.435502 - ComfyUI version: 0.3.34
2025-05-22T19:29:51.477901 - [Prompt Server] web root: C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\web_custom_versions\desktop_app
2025-05-22T19:29:53.802071 - SoX could not be found!

    If you do not have SoX, proceed here:
     - - - http://sox.sourceforge.net/ - - -

    If you do (or think that you should) have SoX, double-check your
    path variables.
    
2025-05-22T19:29:53.813042 - C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\pydub\utils.py:170: RuntimeWarning: Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work
  warn("Couldn't find ffmpeg or avconv - defaulting to ffmpeg, but may not work", RuntimeWarning)
2025-05-22T19:29:53.815036 - Error: File not found - extra_help_file.yaml2025-05-22T19:29:53.815036 - 
2025-05-22T19:29:53.815036 - No customize audios loading path found, use default path.2025-05-22T19:29:53.815036 - 
2025-05-22T19:29:53.894499 - [92m[Allor][0m: 0 nodes were overridden.2025-05-22T19:29:53.894499 - 
2025-05-22T19:29:55.152517 - [92m[Allor][0m: 12 modules were enabled.2025-05-22T19:29:55.152517 - 
2025-05-22T19:29:55.152517 - [92m[Allor][0m: 98 nodes were loaded.2025-05-22T19:29:55.152517 - 
2025-05-22T19:29:55.175582 - Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\nodes.py", line 2131, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-anyline\__init__.py", line 1, in <module>
    from .anyline import AnyLine
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-anyline\anyline.py", line 8, in <module>
    from custom_nodes.comfyui_controlnet_aux.utils import common_annotator_call
ModuleNotFoundError: No module named 'custom_nodes.comfyui_controlnet_aux'

2025-05-22T19:29:55.175582 - Cannot import C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-anyline module for custom nodes: No module named 'custom_nodes.comfyui_controlnet_aux'
2025-05-22T19:29:55.176579 - Adding2025-05-22T19:29:55.176579 -  2025-05-22T19:29:55.176579 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes2025-05-22T19:29:55.176579 -  2025-05-22T19:29:55.176579 - to sys.path2025-05-22T19:29:55.176579 - 
2025-05-22T19:29:55.176579 - Adding2025-05-22T19:29:55.176579 -  2025-05-22T19:29:55.176579 - C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\custom_nodes2025-05-22T19:29:55.176579 -  2025-05-22T19:29:55.176579 - to sys.path2025-05-22T19:29:55.176579 - 
2025-05-22T19:29:55.219485 - Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\nodes.py", line 2131, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-art-venture\__init__.py", line 13, in <module>
    from .modules.nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-art-venture\modules\nodes.py", line 12, in <module>
    from .utility_nodes import (
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-art-venture\modules\utility_nodes.py", line 17, in <module>
    from .utils import pil2tensor, tensor2pil, ensure_package, get_dict_attribute
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-art-venture\modules\utils.py", line 9, in <module>
    import pkg_resources
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\pkg_resources\__init__.py", line 2191, in <module>
    register_finder(pkgutil.ImpImporter, find_on_path)
                    ^^^^^^^^^^^^^^^^^^^
AttributeError: module 'pkgutil' has no attribute 'ImpImporter'. Did you mean: 'zipimporter'?

2025-05-22T19:29:55.219485 - Cannot import C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-art-venture module for custom nodes: module 'pkgutil' has no attribute 'ImpImporter'
2025-05-22T19:29:55.245027 - [Crystools [0;32mINFO[0m] Crystools version: 1.22.1
2025-05-22T19:29:55.270960 - [Crystools [0;32mINFO[0m] CPU: 11th Gen Intel(R) Core(TM) i7-11700KF @ 3.60GHz - Arch: AMD64 - OS: Windows 11
2025-05-22T19:29:55.278938 - [Crystools [0;32mINFO[0m] Pynvml (Nvidia) initialized.
2025-05-22T19:29:55.279936 - [Crystools [0;32mINFO[0m] GPU/s:
2025-05-22T19:29:55.294908 - [Crystools [0;32mINFO[0m] 0) NVIDIA GeForce RTX 3090
2025-05-22T19:29:55.294908 - [Crystools [0;32mINFO[0m] NVIDIA Driver: 572.61
2025-05-22T19:29:55.728832 - [34m[ComfyUI-Easy-Use] server: [0mv1.3.1 [92mLoaded[0m2025-05-22T19:29:55.728832 - 
2025-05-22T19:29:55.728832 - [34m[ComfyUI-Easy-Use] web root: [0mC:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-easy-use\web_version/v2 [92mLoaded[0m2025-05-22T19:29:55.728832 - 
2025-05-22T19:29:55.784190 - ComfyUI-GGUF: Partial torch compile only, consider updating pytorch
2025-05-22T19:29:56.162613 - ### Loading: ComfyUI-Impact-Pack (V8.15.1)2025-05-22T19:29:56.162613 - 
2025-05-22T19:29:56.193650 - [Impact Pack] Wildcards loading done.2025-05-22T19:29:56.193650 - 
2025-05-22T19:29:56.207612 - Total VRAM 24576 MB, total RAM 130953 MB
2025-05-22T19:29:56.207612 - pytorch version: 2.6.0+cu126
2025-05-22T19:29:56.208609 - Set vram state to: NORMAL_VRAM
2025-05-22T19:29:56.208609 - Device: cuda:0 NVIDIA GeForce RTX 3090 : cudaMallocAsync
2025-05-22T19:29:56.488711 - --------------
2025-05-22T19:29:56.488711 - [91m ### Mixlab Nodes: [93mLoaded
2025-05-22T19:29:56.495693 - json_repair## OK2025-05-22T19:29:56.495693 - 
2025-05-22T19:29:56.498685 - ChatGPT.available True
2025-05-22T19:29:56.499682 - edit_mask.available True
2025-05-22T19:29:56.752250 - ## clip_interrogator_model not found: C:\Users\Administrator\Documents\ComfyUI\models\clip_interrogator\Salesforce\blip-image-captioning-base, pls download from https://huggingface.co/Salesforce/blip-image-captioning-base2025-05-22T19:29:56.752250 - 
2025-05-22T19:29:56.752250 - ClipInterrogator.available True
2025-05-22T19:29:56.791236 - ## text_generator_model not found: C:\Users\Administrator\Documents\ComfyUI\models\prompt_generator\text2image-prompt-generator, pls download from https://huggingface.co/succinctly/text2image-prompt-generator/tree/main2025-05-22T19:29:56.791236 - 
2025-05-22T19:29:56.791236 - ## zh_en_model not found: C:\Users\Administrator\Documents\ComfyUI\models\prompt_generator\opus-mt-zh-en, pls download from https://huggingface.co/Helsinki-NLP/opus-mt-zh-en/tree/main2025-05-22T19:29:56.791236 - 
2025-05-22T19:29:56.791236 - PromptGenerate.available True
2025-05-22T19:29:56.791236 - ChinesePrompt.available True
2025-05-22T19:29:56.791236 - RembgNode_.available True
2025-05-22T19:29:56.794228 - ffmpeg could not be found. Using ffmpeg from imageio-ffmpeg.2025-05-22T19:29:56.794228 - 
2025-05-22T19:29:56.857618 - TripoSR.available
2025-05-22T19:29:56.857618 - MiniCPMNode.available
2025-05-22T19:29:56.887602 - Scenedetect.available
2025-05-22T19:29:56.956525 - FishSpeech.available
2025-05-22T19:29:56.962509 - SenseVoice.available
2025-05-22T19:29:56.966499 - Whisper.available False
2025-05-22T19:29:56.967496 - fal-client## OK2025-05-22T19:29:56.967496 - 
2025-05-22T19:29:56.975637 - FalVideo.available
2025-05-22T19:29:56.975637 - [93m -------------- [0m
2025-05-22T19:29:57.756093 - Failed to auto update `Quality of Life Suit` 2025-05-22T19:29:57.756093 - 
2025-05-22T19:29:57.757091 - [33mQualityOfLifeSuit_Omar92_DIR:[0m C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-QualityOfLifeSuit_Omar922025-05-22T19:29:57.757091 - 
2025-05-22T19:29:58.021461 - no module 'xformers'. Processing without...2025-05-22T19:29:58.021461 - 
2025-05-22T19:29:58.022459 - no module 'xformers'. Processing without...2025-05-22T19:29:58.022459 - 
2025-05-22T19:29:58.267568 - Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\nodes.py", line 2131, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-UNO\__init__.py", line 1, in <module>
    from .nodes import ImagePathLoader, UNOParams, UNOGenerator, ImageConcat, ImageSave, ConfigSave
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-UNO\nodes.py", line 10, in <module>
    from .src.flux.pipeline import UNOPipeline, preprocess_ref
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-UNO\src\flux\pipeline.py", line 24, in <module>
    from uno.flux.modules.layers import (
ModuleNotFoundError: No module named 'uno'

2025-05-22T19:29:58.267568 - Cannot import C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-UNO module for custom nodes: No module named 'uno'
2025-05-22T19:29:58.293587 - Warning: Could not load sageattention: No module named 'sageattention'2025-05-22T19:29:58.293587 - 
2025-05-22T19:29:58.293587 - sageattention package is not installed2025-05-22T19:29:58.293587 - 
2025-05-22T19:29:58.300568 - Python version is above 3.10, patching the collections module.2025-05-22T19:29:58.300568 - 
2025-05-22T19:29:58.302564 - C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\transformers\models\auto\image_processing_auto.py:604: FutureWarning: The image_processor_class argument is deprecated and will be removed in v4.42. Please use `slow_image_processor_class`, or `fast_image_processor_class` instead
  warnings.warn(
2025-05-22T19:29:58.341084 - ------------------------------------------2025-05-22T19:29:58.341084 - 
2025-05-22T19:29:58.341084 - [34mComfyroll Studio v1.76 : [92m 175 Nodes Loaded[0m2025-05-22T19:29:58.341084 - 
2025-05-22T19:29:58.341084 - ------------------------------------------2025-05-22T19:29:58.341084 - 
2025-05-22T19:29:58.341084 - ** For changes, please see patch notes at https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes/blob/main/Patch_Notes.md2025-05-22T19:29:58.341084 - 
2025-05-22T19:29:58.341084 - ** For help, please see the wiki at https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes/wiki2025-05-22T19:29:58.341084 - 
2025-05-22T19:29:58.341084 - ------------------------------------------2025-05-22T19:29:58.341084 - 
2025-05-22T19:29:58.345074 - [36;20m[C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_controlnet_aux] | INFO -> Using ckpts path: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_controlnet_aux\ckpts[0m
2025-05-22T19:29:58.345074 - [36;20m[C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_controlnet_aux] | INFO -> Using symlinks: False[0m
2025-05-22T19:29:58.345074 - [36;20m[C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_controlnet_aux] | INFO -> Using ort providers: ['CUDAExecutionProvider', 'DirectMLExecutionProvider', 'OpenVINOExecutionProvider', 'ROCMExecutionProvider', 'CPUExecutionProvider', 'CoreMLExecutionProvider'][0m
2025-05-22T19:29:58.359036 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_controlnet_aux\node_wrappers\dwpose.py:26: UserWarning: DWPose: Onnxruntime not found or doesn't come with acceleration providers, switch to OpenCV with CPU device. DWPose might run very slowly
  warnings.warn("DWPose: Onnxruntime not found or doesn't come with acceleration providers, switch to OpenCV with CPU device. DWPose might run very slowly")
2025-05-22T19:29:58.923739 - C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\albumentations\__init__.py:13: UserWarning: A new version of Albumentations is available: 2.0.7 (you have 1.4.15). Upgrade using: pip install -U albumentations. To disable automatic update checks, set the environment variable NO_ALBUMENTATIONS_UPDATE to 1.
  check_for_updates()
2025-05-22T19:29:59.018789 - C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\timm\models\layers\__init__.py:48: FutureWarning: Importing from timm.models.layers is deprecated, please import via timm.layers
  warnings.warn(f"Importing from {__name__} is deprecated, please import via timm.layers", FutureWarning)
2025-05-22T19:29:59.069167 - Currently enabled native sdp backends:2025-05-22T19:29:59.069167 -  2025-05-22T19:29:59.069167 - ['flash', 'math', 'mem_efficient', 'cudnn']2025-05-22T19:29:59.069167 - 
2025-05-22T19:29:59.070164 - Xformers is not installed!2025-05-22T19:29:59.070164 - 
2025-05-22T19:29:59.070164 - Flash Attn is not installed!2025-05-22T19:29:59.070164 - 
2025-05-22T19:29:59.071162 - Sage Attn is not installed!2025-05-22T19:29:59.071162 - 
2025-05-22T19:29:59.142539 - [92m[tinyterraNodes] [32mLoaded[0m2025-05-22T19:29:59.142539 - 
2025-05-22T19:29:59.232908 - 
[36mEfficiency Nodes:[0m Attempting to add Control Net options to the 'HiRes-Fix Script' Node (comfyui_controlnet_aux add-on)...[92mSuccess![0m2025-05-22T19:29:59.232908 - 
2025-05-22T19:29:59.247869 - 
2025-05-22T19:29:59.247869 - [92m[rgthree-comfy] Loaded 42 exciting nodes. 🎉[00m2025-05-22T19:29:59.247869 - 
2025-05-22T19:29:59.247869 - 
2025-05-22T19:29:59.950541 - Searge-SDXL v4.3.1 in C:\Users\Administrator\Documents\ComfyUI\custom_nodes\SeargeSDXL2025-05-22T19:29:59.950541 - 
2025-05-22T19:30:00.609875 - [34mWAS Node Suite: [0mOpenCV Python FFMPEG support is enabled[0m2025-05-22T19:30:00.609875 - 
2025-05-22T19:30:00.609875 - [34mWAS Node Suite [93mWarning: [0m`ffmpeg_bin_path` is not set in `C:\Users\Administrator\Documents\ComfyUI\custom_nodes\was-node-suite-comfyui\was_suite_config.json` config file. Will attempt to use system ffmpeg binaries if available.[0m2025-05-22T19:30:00.609875 - 
2025-05-22T19:30:01.207554 - [34mWAS Node Suite: [0mFinished.[0m [32mLoaded[0m [0m220[0m [32mnodes successfully.[0m2025-05-22T19:30:01.207554 - 
2025-05-22T19:30:01.207554 - 
	[3m[93m"Believe in yourself, and the rest will fall into place."[0m[3m - Unknown[0m
2025-05-22T19:30:01.207554 - 
2025-05-22T19:30:01.215532 - ### Loading: ComfyUI-Manager (V3.30.4)
2025-05-22T19:30:01.215532 - [ComfyUI-Manager] network_mode: public
2025-05-22T19:30:01.216530 - ### ComfyUI Revision: UNKNOWN (The currently installed ComfyUI is not a Git repository)
2025-05-22T19:30:01.250454 - 
Import times for custom nodes:
2025-05-22T19:30:01.250454 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_RH_UNO
2025-05-22T19:30:01.250932 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\joycaption_comfyui
2025-05-22T19:30:01.250932 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ICEF
2025-05-22T19:30:01.250932 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_PRNodes
2025-05-22T19:30:01.250932 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mxtoolkit
2025-05-22T19:30:01.251434 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_AdvancedRefluxControl
2025-05-22T19:30:01.251434 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-Align
2025-05-22T19:30:01.251434 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\cg-use-everywhere
2025-05-22T19:30:01.251434 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_ttp_toolset
2025-05-22T19:30:01.251434 -    0.0 seconds: C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\custom_nodes\websocket_image_save.py
2025-05-22T19:30:01.251434 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-Copilot
2025-05-22T19:30:01.251851 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-in-context-lora-utils
2025-05-22T19:30:01.251851 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-inpainteasy
2025-05-22T19:30:01.251851 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-GGUF
2025-05-22T19:30:01.251851 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-post-processing-nodes
2025-05-22T19:30:01.252354 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-automaticcfg
2025-05-22T19:30:01.252354 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_zenid
2025-05-22T19:30:01.252354 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_mienodes
2025-05-22T19:30:01.252354 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-inpaint-nodes
2025-05-22T19:30:01.252354 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-QualityOfLifeSuit_Omar92
2025-05-22T19:30:01.252354 -    0.0 seconds (IMPORT FAILED): C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-UNO
2025-05-22T19:30:01.252354 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_essentials
2025-05-22T19:30:01.252854 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR
2025-05-22T19:30:01.252854 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-florence2
2025-05-22T19:30:01.252854 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-various
2025-05-22T19:30:01.252854 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-Chibi-Nodes
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_ultimatesdupscale
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\derfuu_comfyui_moddednodes
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfy-image-saver
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-WanVideoWrapper
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_RH_FramePack
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-custom-scripts
2025-05-22T19:30:01.253358 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-NeuralMedia
2025-05-22T19:30:01.253859 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\rgthree-comfy
2025-05-22T19:30:01.253859 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-videohelpersuite
2025-05-22T19:30:01.253859 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-nunchaku
2025-05-22T19:30:01.253859 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_caption_this
2025-05-22T19:30:01.254362 -    0.0 seconds (IMPORT FAILED): C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-anyline
2025-05-22T19:30:01.254362 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_Comfyroll_CustomNodes
2025-05-22T19:30:01.254362 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-KJNodes
2025-05-22T19:30:01.254362 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\efficiency-nodes-comfyui
2025-05-22T19:30:01.254866 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-impact-pack
2025-05-22T19:30:01.254866 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_controlnet_aux
2025-05-22T19:30:01.254866 -    0.0 seconds: C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\custom_nodes\ComfyUI-Manager
2025-05-22T19:30:01.255369 -    0.0 seconds (IMPORT FAILED): C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-art-venture
2025-05-22T19:30:01.255369 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_LayerStyle_Advance
2025-05-22T19:30:01.255369 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_layerstyle
2025-05-22T19:30:01.255892 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-fluxtrainer
2025-05-22T19:30:01.255892 -    0.0 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_tinyterranodes
2025-05-22T19:30:01.255892 -    0.1 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-tensorops
2025-05-22T19:30:01.255892 -    0.1 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_segment_anything
2025-05-22T19:30:01.255892 -    0.1 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-Crystools
2025-05-22T19:30:01.255892 -    0.1 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-easy-use
2025-05-22T19:30:01.255892 -    0.1 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\audiotools
2025-05-22T19:30:01.255892 -    0.2 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-tooling-nodes
2025-05-22T19:30:01.255892 -    0.2 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-kynode
2025-05-22T19:30:01.255892 -    0.3 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-supir
2025-05-22T19:30:01.256891 -    0.4 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-detail-daemon
2025-05-22T19:30:01.256891 -    0.4 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-hunyan3dwrapper
2025-05-22T19:30:01.256891 -    0.5 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes
2025-05-22T19:30:01.256891 -    0.6 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui_instantid
2025-05-22T19:30:01.256891 -    0.7 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\SeargeSDXL
2025-05-22T19:30:01.256891 -    0.7 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-ollama
2025-05-22T19:30:01.256891 -    1.2 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\was-node-suite-comfyui
2025-05-22T19:30:01.256891 -    1.3 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI-Allor
2025-05-22T19:30:01.256891 -    1.5 seconds: C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ace-step
2025-05-22T19:30:01.256891 - 
2025-05-22T19:30:01.280828 - Starting server

2025-05-22T19:30:01.280828 - To see the GUI go to: http://127.0.0.1:8002
2025-05-22T19:30:01.467482 - [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
2025-05-22T19:30:01.503004 - [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
2025-05-22T19:30:01.566972 - [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
2025-05-22T19:30:01.614379 - [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
2025-05-22T19:30:01.646428 - [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
2025-05-22T19:30:03.813956 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe-lightbox.esm.min.js2025-05-22T19:30:03.814686 - 
2025-05-22T19:30:03.874649 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/photoswipe.min.css2025-05-22T19:30:03.874649 - 
2025-05-22T19:30:03.923191 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/pickr.min.js2025-05-22T19:30:03.924419 - 
2025-05-22T19:30:03.985927 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/model-viewer.min.js2025-05-22T19:30:03.985927 - 
2025-05-22T19:30:03.990422 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/classic.min.css2025-05-22T19:30:03.990422 - 
2025-05-22T19:30:04.020947 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.css2025-05-22T19:30:04.020947 - 
2025-05-22T19:30:04.022449 - C:\Users\Administrator\Documents\ComfyUI\custom_nodes\comfyui-mixlab-nodes\webApp\lib/juxtapose.min.js2025-05-22T19:30:04.022449 - 
2025-05-22T19:30:04.273432 - [ERROR] An error occurred while retrieving information for the 'ACEModelLoader' node.
2025-05-22T19:30:04.274429 - Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\server.py", line 595, in get_object_info
    out[x] = node_info(x)
             ^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\server.py", line 559, in node_info
    info['input'] = obj_class.INPUT_TYPES()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ace-step\ace_step_nodes.py", line 263, in INPUT_TYPES
    models = [name for name in os.listdir(model_path) if os.path.isdir(os.path.join(model_path, name)) and not name.startswith(".")]
                               ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] 系统找不到指定的路径。: 'C:\\Users\\Administrator\\Documents\\ComfyUI\\models\\TTS\\ACE-Step-v1-3.5B'

2025-05-22T19:30:04.326757 - [33mQualityOfLifeSuit_Omar92:[0m:NSP ready2025-05-22T19:30:04.326757 - 
2025-05-22T19:30:06.130230 - [ERROR] An error occurred while retrieving information for the 'ACEModelLoader' node.
2025-05-22T19:30:06.131229 - Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\server.py", line 595, in get_object_info
    out[x] = node_info(x)
             ^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\server.py", line 559, in node_info
    info['input'] = obj_class.INPUT_TYPES()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ace-step\ace_step_nodes.py", line 263, in INPUT_TYPES
    models = [name for name in os.listdir(model_path) if os.path.isdir(os.path.join(model_path, name)) and not name.startswith(".")]
                               ^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] 系统找不到指定的路径。: 'C:\\Users\\Administrator\\Documents\\ComfyUI\\models\\TTS\\ACE-Step-v1-3.5B'

2025-05-22T19:30:06.465721 - FETCH ComfyRegistry Data: 5/862025-05-22T19:30:06.465721 - 
2025-05-22T19:30:11.648866 - FETCH ComfyRegistry Data: 10/862025-05-22T19:30:11.648866 - 
2025-05-22T19:30:13.666131 - got prompt
2025-05-22T19:30:15.923167 - 
Loading pipeline components...: 100%|██████████| 5/5 [00:01<00:00,  3.88it/s]2025-05-22T19:30:15.923167 - 
Loading pipeline components...: 100%|██████████| 5/5 [00:01<00:00,  3.42it/s]2025-05-22T19:30:15.923167 - 
2025-05-22T19:30:15.923167 - You have disabled the safety checker for <class 'diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline'> by passing `safety_checker=None`. Ensure that you abide to the conditions of the Stable Diffusion license and do not expose unfiltered results in services or applications open to the public. Both the diffusers team and Hugging Face strongly recommend to keep the safety filter enabled in all public facing circumstances, disabling it only for use-cases that involve analyzing network behavior or auditing its results. For more information, please have a look at https://github.com/huggingface/diffusers/pull/254 .
2025-05-22T19:30:15.925199 - You have disabled the safety checker for <class 'C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR.comfyui_invsr_trimmed.pipeline_stable_diffusion_inversion_sr.StableDiffusionInvEnhancePipeline'> by passing `safety_checker=None`. Ensure that you abide to the conditions of the Stable Diffusion license and do not expose unfiltered results in services or applications open to the public. Both the diffusers team and Hugging Face strongly recommend to keep the safety filter enabled in all public facing circumstances, disabling it only for use-cases that involve analyzing network behavior or auditing its results. For more information, please have a look at https://github.com/huggingface/diffusers/pull/254 .
2025-05-22T19:30:17.338404 - Activating gradient checkpoing for vae...2025-05-22T19:30:17.338404 - 
2025-05-22T19:30:17.577975 - !!! Exception during processing !!! maximum recursion depth exceeded while getting the str of an object
2025-05-22T19:30:17.591953 - Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 349, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 224, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 196, in _map_node_over_list
    process_inputs(input_dict, i)
  File "C:\Users\Administrator\AppData\Local\Programs\@comfyorgcomfyui-electron\resources\ComfyUI\execution.py", line 185, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR\node.py", line 95, in loadmodel
    base_sampler = BaseSampler(configs)
                   ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR\comfyui_invsr_trimmed\sampler_invsr.py", line 42, in __init__
    self.build_model()
  File "C:\Users\Administrator\Documents\ComfyUI\custom_nodes\ComfyUI_InvSR\comfyui_invsr_trimmed\sampler_invsr.py", line 90, in build_model
    sd_pipe.vae._set_gradient_checkpointing(sd_pipe.vae.encoder, True)
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\diffusers\models\modeling_utils.py", line 1718, in _set_gradient_checkpointing
    logger.debug(f"Setting `gradient_checkpointing={enable}` for '{name}'")
                                                   ^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^
  [Previous line repeated 740 more times]
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\container.py", line 369, in __repr__
    list_of_reprs = [repr(item) for item in self]
                     ^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\container.py", line 369, in __repr__
    list_of_reprs = [repr(item) for item in self]
                     ^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2940, in __repr__
    mod_str = repr(module)
              ^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\module.py", line 2934, in __repr__
    extra_repr = self.extra_repr()
                 ^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\ComfyUI\.venv\Lib\site-packages\torch\nn\modules\normalization.py", line 316, in extra_repr
    return "{num_groups}, {num_channels}, eps={eps}, " "affine={affine}".format(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RecursionError: maximum recursion depth exceeded while getting the str of an object

2025-05-22T19:30:17.604918 - Prompt executed in 3.94 seconds

```
## Attached Workflow
Please make sure that workflow does not contain any sensitive information such as API keys or passwords.
```
{"id":"f98ada92-61c6-4d94-9b4d-b6b21d7066c8","revision":0,"last_node_id":28,"last_link_id":42,"nodes":[{"id":26,"type":"PreviewImage","pos":[35.55903625488281,643.07470703125],"size":[889.1287231445312,869.5750732421875],"flags":{},"order":4,"mode":0,"inputs":[{"localized_name":"images","name":"images","type":"IMAGE","link":39}],"outputs":[],"properties":{"cnr_id":"comfy-core","ver":"0.3.34","Node name for S&R":"PreviewImage","widget_ue_connectable":{}},"widgets_values":[]},{"id":17,"type":"LoadImage","pos":[-1105.5140380859375,1002.91455078125],"size":[367.18212890625,439.6049499511719],"flags":{},"order":0,"mode":0,"inputs":[{"localized_name":"image","name":"image","type":"COMBO","widget":{"name":"image"},"link":null},{"localized_name":"choose file to upload","name":"upload","type":"IMAGEUPLOAD","widget":{"name":"upload"},"link":null}],"outputs":[{"localized_name":"IMAGE","name":"IMAGE","type":"IMAGE","slot_index":0,"links":[41]},{"localized_name":"MASK","name":"MASK","type":"MASK","links":null}],"properties":{"cnr_id":"comfy-core","ver":"0.3.34","Node name for S&R":"LoadImage","widget_ue_connectable":{}},"widgets_values":["Contrasted-Retouched.png","image"]},{"id":28,"type":"ImageResizeKJ","pos":[-670.8065185546875,989.8233032226562],"size":[270,238],"flags":{},"order":2,"mode":0,"inputs":[{"localized_name":"image","name":"image","type":"IMAGE","link":41},{"localized_name":"width_input","name":"width_input","shape":7,"type":"INT","link":null},{"localized_name":"height_input","name":"height_input","shape":7,"type":"INT","link":null},{"localized_name":"get_image_size","name":"get_image_size","shape":7,"type":"IMAGE","link":null},{"localized_name":"width","name":"width","type":"INT","widget":{"name":"width"},"link":null},{"localized_name":"height","name":"height","type":"INT","widget":{"name":"height"},"link":null},{"localized_name":"upscale_method","name":"upscale_method","type":"COMBO","widget":{"name":"upscale_method"},"link":null},{"localized_name":"keep_proportion","name":"keep_proportion","type":"BOOLEAN","widget":{"name":"keep_proportion"},"link":null},{"localized_name":"divisible_by","name":"divisible_by","type":"INT","widget":{"name":"divisible_by"},"link":null},{"localized_name":"crop","name":"crop","shape":7,"type":"COMBO","widget":{"name":"crop"},"link":null}],"outputs":[{"localized_name":"IMAGE","name":"IMAGE","type":"IMAGE","links":[42]},{"localized_name":"width","name":"width","type":"INT","links":null},{"localized_name":"height","name":"height","type":"INT","links":null}],"properties":{"cnr_id":"comfyui-kjnodes","ver":"a5bd3c86c8ed6b83c55c2d0e7a59515b15a0137f","Node name for S&R":"ImageResizeKJ","widget_ue_connectable":{}},"widgets_values":[1024,1024,"nearest-exact",false,2,"disabled"]},{"id":25,"type":"LoadInvSRModels","pos":[-877.6565551757812,719.943603515625],"size":[413.84686279296875,130],"flags":{},"order":1,"mode":0,"inputs":[{"localized_name":"sd_model","name":"sd_model","type":"COMBO","widget":{"name":"sd_model"},"link":null},{"localized_name":"invsr_model","name":"invsr_model","type":"COMBO","widget":{"name":"invsr_model"},"link":null},{"localized_name":"dtype","name":"dtype","type":"COMBO","widget":{"name":"dtype"},"link":null},{"localized_name":"tiled_vae","name":"tiled_vae","type":"BOOLEAN","widget":{"name":"tiled_vae"},"link":null}],"outputs":[{"localized_name":"invsr_pipe","name":"invsr_pipe","type":"INVSR_PIPE","slot_index":0,"links":[38]}],"properties":{"cnr_id":"ComfyUI_InvSR","ver":"9ea864ee537b3d4afc01a2753db83bfd59d7a59d","Node name for S&R":"LoadInvSRModels","widget_ue_connectable":{}},"widgets_values":["stabilityai/sd-turbo","noise_predictor_sd_turbo_v5.pth","bf16",true]},{"id":27,"type":"InvSRSampler","pos":[-365.7756652832031,859.0978393554688],"size":[315,246],"flags":{},"order":3,"mode":0,"inputs":[{"localized_name":"invsr_pipe","name":"invsr_pipe","type":"INVSR_PIPE","link":38},{"localized_name":"images","name":"images","type":"IMAGE","link":42},{"localized_name":"num_steps","name":"num_steps","type":"INT","widget":{"name":"num_steps"},"link":null},{"localized_name":"cfg","name":"cfg","type":"FLOAT","widget":{"name":"cfg"},"link":null},{"localized_name":"batch_size","name":"batch_size","type":"INT","widget":{"name":"batch_size"},"link":null},{"localized_name":"chopping_batch_size","name":"chopping_batch_size","type":"INT","widget":{"name":"chopping_batch_size"},"link":null},{"localized_name":"chopping_size","name":"chopping_size","type":"COMBO","widget":{"name":"chopping_size"},"link":null},{"localized_name":"color_fix","name":"color_fix","type":"COMBO","widget":{"name":"color_fix"},"link":null},{"localized_name":"seed","name":"seed","type":"INT","widget":{"name":"seed"},"link":null}],"outputs":[{"localized_name":"image","name":"image","type":"IMAGE","slot_index":0,"links":[39]}],"properties":{"cnr_id":"ComfyUI_InvSR","ver":"9ea864ee537b3d4afc01a2753db83bfd59d7a59d","Node name for S&R":"InvSRSampler","widget_ue_connectable":{}},"widgets_values":[5,5,1,8,128,"wavelet",2458357809,"randomize"]}],"links":[[38,25,0,27,0,"INVSR_PIPE"],[39,27,0,26,0,"IMAGE"],[41,17,0,28,0,"IMAGE"],[42,28,0,27,1,"IMAGE"]],"groups":[],"config":{},"extra":{"ds":{"scale":0.7972024500000005,"offset":[2675.203085037118,-121.97803352697542]},"VHS_latentpreview":false,"VHS_latentpreviewrate":0,"ue_links":[],"links_added_by_ue":[]},"version":0.4}
```

## Additional Context
(Please add any additional context or steps to reproduce the error here)

--- a/comfyui_invsr_trimmed/sampler_invsr.py
+++ b/comfyui_invsr_trimmed/sampler_invsr.py
@@ -87,8 +87,7 @@ class BaseSampler:
             sd_pipe.vae.tile_sample_min_size = self.configs.sample_tiled_size
         if self.configs.gradient_checkpointing_vae:
             self.write_log(f"Activating gradient checkpoing for vae...")
-            sd_pipe.vae._set_gradient_checkpointing()
-            sd_pipe.vae._set_gradient_checkpointing()
+            sd_pipe.vae.enable_gradient_checkpointing()
 
         model_configs = self.configs.model_start
         params = model_configs.get('params', dict)

--- a/comfyui_invsr_trimmed/sampler_invsr.py
+++ b/comfyui_invsr_trimmed/sampler_invsr.py
@@ -87,8 +87,8 @@ class BaseSampler:
             sd_pipe.vae.tile_sample_min_size = self.configs.sample_tiled_size
         if self.configs.gradient_checkpointing_vae:
             self.write_log(f"Activating gradient checkpoing for vae...")
-            sd_pipe.vae._set_gradient_checkpointing(sd_pipe.vae.encoder, True)
-            sd_pipe.vae._set_gradient_checkpointing(sd_pipe.vae.decoder, True)
+            sd_pipe.vae._set_gradient_checkpointing()
+            sd_pipe.vae._set_gradient_checkpointing()
 
         model_configs = self.configs.model_start
         params = model_configs.get('params', dict)

